src/components/home/ProjectCard.tsx

```diff
@@ -0,0 +1,31 @@
+import Image, { StaticImageData } from "next/image";
+
+interface ProjectCardProps {
+  backgroundImage?: StaticImageData;
+  description: string;
+  title: string;
+}
+
+const ProjectCard: React.FC<ProjectCardProps> = ({
+  backgroundImage,
+  description,
+  title,
+}) => {
+  return (
+    <div className="flex flex-col items-center rounded-2xl border-4 border-devstation-project-card-border-green bg-devstation-project-card-bg-green p-5">
+      <div className="mb-2 flex w-3/4 items-end justify-end rounded-2xl">
+        {backgroundImage && (
+          <Image src={backgroundImage} alt={title} className="rounded-2xl" />
+        )}
+
+        <div className="absolute flex items-end justify-end p-[2px_20px] text-right uppercase">
```

Comment:
instead of p-[2px-20px] use the tailwind classes to manipulate padding for x axis, y axis, etc. or left/right/bottom/top. you can do so with px/py/pt/pb/pl/pr

Reply:
Did it with px and py